Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentImportData.swift

```diff
@@ -0,0 +1,24 @@
+// Copyright 2024 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import ArcGIS
+import OSLog
+import SwiftUI
+
+/// Data used to create the attachment.
+struct AttachmentImportData: Equatable {
+    var data: Data
+    var contentType: String
```

Comment:
Should the first two properties be constants?

```suggestion
    let data: Data
    let contentType: String
```